src/core/targeting/youtube-ima.ts

```diff
@@ -85,6 +85,9 @@ const buildImaAdTagUrl = ({
 		impl: 's',
 		vad_type: 'linear',
 		vpos: 'preroll',
+		description_url: encodeURIComponent(
+			`https://www.theguardian.com/${window.guardian.config.page.pageId}`,
```

Comment:
I wonder if we can combine `window.guardian.config.page.host` to get the URL domain from config

```suggestion
			`${window.guardian.config.page.host}/${window.guardian.config.page.pageId}`,
```